ra_aid/__main__.py

```
@@ -788,6 +806,130 @@ def build_status():
+        if args.expert_provider and args.expert_model:
+            expert_llm = initialize_llm(
+                args.expert_provider, args.expert_model, temperature=args.temperature
+            )
+        
+        # Get custom tools
+        custom_tools = get_custom_tools(mcp_use_client=mcp_use_client_instance)
+        
+        # Create the planning agent
+        planning_agent = create_agent(
+            llm,
+            get_planning_tools(
```

Ruff (0.8.2)

Undefined name `get_planning_tools`

(F821)